Formula/logrotate.rb

```diff
@@ -63,6 +65,21 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    # Generate a test log file of size >= 1byte
+    (testpath/"test.log").write("testlograndomstring")
+    # Logrotate config file which rotates the log file of size >= 1 byte
+    (testpath/"testlogrotate.conf").write <<-EOS.undent
+        #{testpath}/test.log {
+        size 1
+        copytruncate
+    }
+    EOS
+    # Rotate the log file and check if the new file's size is 0 bytes
```

Comment:
Can remove all the comments above, they are good but pretty self-explanatory. Thanks!